CMakeLists.txt

```diff
@@ -177,10 +209,14 @@ source_group(pdextra    FILES ${PD_EXTRA_SOURCES})
 source_group(libpd      FILES ${LIBPD_SOURCES})
 source_group(libpdutils FILES ${LIBPD_UTILS_SOURCES})
 
-include_directories(libpd_wrapper)
-include_directories(pure-data/src)
-
-find_package(Threads REQUIRED)
+# set the output library name for libpd depending on the settings
+set(LIBPD_OUTPUT_NAME     pd)
+if (WIN32)
+    set(LIBPD_OUTPUT_NAME libpd)
+endif()
+if (PD_MULTI)
+    set(LIBPD_OUTPUT_NAME ${LIBPD_OUTPUT_NAME}-multi)
```

Comment:
I see your reasoning for the `-multi` suffix, but it is not canonical. The makefile build system does not do this. Also, this breaks on Windows because externals always link against `pd.dll`.

Reply:
hmm I think I just followed what was there in the previous version of the CMakeLists

Reply:
I see! See my other comment https://github.com/libpd/libpd/pull/383#discussion_r1442268113